ptero_lsf/api/v1/views.py

```diff
@@ -1,24 +1,30 @@
 from . import validators
-from flask import g, url_for
+from flask import g, request, url_for
 from flask.ext.restful import Resource
 from ptero_common.logging_configuration import logged_response
 import logging
+import uuid
 
 
 LOG = logging.getLogger(__name__)
 
 
 class JobListView(Resource):
-    @logged_response(logger=LOG)
     def post(self):
+        request.id = uuid.uuid4()
```

Comment:
Instead of creating and logging a uuid for the request, we can create a uuid here to represent the Job, and call it job_id.  We can log the job_id here, and pass through to the backend where we create and persist the Job object, instead of depending on Job to have a default for the job_id column.